### ISSUE TITLE
Installation instructions

### ISSUE THREAD

**Issue by kebasaa** [hobinjk/mqtt-adapter#2]
Hi,

Could you please add some installation instructions? I cloned this repository to /home/pi/.mozilla-iot/addons/, then restarted the gateway using

> $ sudo systemctl restart mozilla-iot-gateway.service

But after restarting the service (note: I'm using IoT Gateway 0.6.1), the MQTT addon was not listed in the addon list. What can I do to get it to work? I have a number of MQTT sensors.

Also, is this an MQTT broker? Ideally, it would be. And finally, thanks a lot for your work!

**Comment by rzr** [comment]
I think it's under progress, and not ready to be used because you can see some hardcoded path in source..

Anyway I can help of building MQTT webthings using iotjs

ping me if interested then I will share details on:
https://github.com/rzr/webthing-iotjs/wiki

**Comment by hobinjk** [comment]
Yes, sorry. What rzr said is correct. This is still very much a work-in-progress. The main difficulty moving forwards with it is developing a way to translate between arbitrary MQTT-based protocols which proved to be more than I could get done

**Comment by elviosebastianelli** [comment]
Hello hobinjk, what Is the current state of the project? Wanting to collaborate from where should I start?

**Comment by rzr** [comment]
I plan to release a (node) mqtt adapter based on this:
https://twitter.com/RzrFreeFr/status/1093553415907655680

**Comment by hobinjk** [comment]
Sorry for the delay in responding, @elviosebastianelli collaborating with rzr is the best option here since I'm occupied with other projects on the gateway.

**Comment by rzr** [comment]
yes @elviosebastianelli  your welcome to ask me about MQTT

I am a bit busy now but I have shared some hints on top of my head:

* https://github.com/TizenTeam/webthing-iotjs/wiki/TODO

Feel free to edit this chapter and document your experiences I will improve it for community.

Regards

**Comment by rzr** [comment]
Relate-to: https://github.com/WebThingsIO/addon-list/issues/1039
